docs/topics/tour/kotlin-tour-hello-world.md

```diff
@@ -0,0 +1,115 @@
+* [`println()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html) and [`print()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/print.html) functions print their arguments to standard output
+
+Functions are discussed in more detail in a couple of chapters. Until then, all examples use the `main` function.
+
+## Variables
+
+All programs need to be able to store data, and variables help you to do just that. In Kotlin, you can declare:
+* read-only variables with `val`
+* mutable variables with `var`
+
+To assign a value, use the assignment operator `=`.
+
+```kotlin
+fun main() { 
+//sampleStart
+    val popcorn = 5
+    val hotdog = 7
+    var customers = 10
+    
+    //Some customers leave the queue
+    customers = 8
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-variables"}
+
+> Variables can be declared at top level as well as within functions.
+{type="info"}
+
+As `customers` is a mutable variable, its value can be reassigned after declaration.
+
+> We recommend that you declare all variables as read-only (`val`) by default. Declare mutable variables (`var`) only if 
+> necessary.
+{type="info"}
+
+## String templates
+
+It's useful to know how to print the contents of variables to standard output. You can do this with **string templates**. 
+You can use template expressions to access data stored in variables and other objects, and convert them into strings.
+A string value is a sequence of characters in double quotes (`"`). Template expressions always start with a dollar sign (`$`).
+
+To evaluate a piece of code in a template expression, place the code within curly braces `{}` after the dollar sign (`$`).
+
+For example:
+
+```kotlin
+fun main() { 
+//sampleStart
+    val customers = 10
+    println("There are $customers customers")
+    //There are 10 customers
+    
+    println("There are ${customers + 1} customers")
+    //There are 11 customers
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-string-templates"}
+
+For more information, see [String templates](strings.md).
+
+You will notice that there aren't any types declared for variables. Kotlin has inferred the type itself: `Int`. This tour
+explains the different Kotlin basic types and how to declare them in the [next chapter](kotlin-tour-basic-types.md).
+
+## Practice
+
+### Exercise {initial-collapse-state="collapsed"}
+Complete the code below to make the program print `"Mary is 20 years old"` to standard output.
+
+|---|---|
+```kotlin
+fun main() {
+    val name = "Mary"
+    val age = 20
+    //Write your code here
+}
+```
+{validate="false" kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-hello-world-exercise"}
+
+|---|---|
+```kotlin
+fun main() {
+    val name = "Mary"
+    val age = 20
+    println("$name is $age years old")
+}
+```
+{initial-collapse-state="collapsed" collapsed-title="Example solution"}
```

Comment:
Even if there would be an arrow, don't we want to add a section "Next step" to mark the next step explicitly?

Reply:
I add a link to the next section in the last sentence of every chapter. Do you think this isn't enough? Using a header with "Next step" doesn't look as nice. But I could ask our designers to design a "Next step" button to make it nicer 🤔 WDYT?